src/app/navbar/navbar.component.html

```diff
@@ -16,6 +16,10 @@
     <div class="collapse navbar-collapse menu" id="navbarSupportedContent" [collapse]="isCollapsed">
         <ul class="navbar-nav mr-auto">
         </ul>
+        <span class="navbar-text">
+            <img src='http://openweathermap.org/img/w/{{icon}}.png' alt='Icon depicting current weather.'>
+        </span>
+        <span class="navbar-text">{{temperature}}°F</span>
```

Comment:
Make it so when you click no the span, it switches between Fahrenheit and Celsius